src/utils/functions/mod.ts

```diff
@@ -28,4 +29,27 @@ export async function CheckPermsBoth(interaction:CommandInteraction,perms:Permis
         return false
     }
     return true
+}
+export async function ModLog(client:DiscordClient,guildID:string,embed:MessageEmbed){
+    var guildOption = client.guildSettings.get(guildID)
+    if(!guildOption?.moderationSettings?.modLogChannel)return
+    var guild = await client.guilds.fetch(guildID)
+    guild.fetchAuditLogs()
```

Comment:
remember fetching audit log is an api call. It's better to not do it for every single update